data/sql/updates/pending_db_world/rev_1640281609265649594.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1640281609265649594');
+
+-- High Warlord Whirlaxis is not skinnable fix. Never was skinnable. Flags Corrected.
+UPDATE `creature_template` SET `unit_flags`='0', `skinloot`='0' WHERE  `entry`=15204;
```

Comment:
why using `'` for numeric values? :D

also, as you're doing this manually, would be better to edit flags this way:

https://www.azerothcore.org/wiki/sql-standards#flags-amp-bits

Reply:
definately was a over look on my end. adjusted to standards.